jetty-http/src/main/java/org/eclipse/jetty/http/HttpGenerator.java

```diff
@@ -67,10 +67,11 @@
     private Boolean _persistent = null;
 
     private final int _send;
+    private final boolean _chunkEOFContent;
```

Comment:
We're ok with it only making its way into 9.4.x if you feel it's cleaner that way.
Just so I understand what you're proposing, would "Transfer-Encoding: chunked" be sent in the request to signal that as the desired response behaviour?